src/Util/StringUtil.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Util;
+
+class StringUtil
```

Comment:
How about `Sonata\BlockBundle\BlockName\ConvertFromFqcn::__invoke` instead, with that class final? That kind of "util" class often ends up growing to the infinite, and being cluttered with unrelated methods whose which have only the `string` => `string` signature in common.